src/WebJobs.Extensions.DocumentDB/Trigger/DocumentDBTriggerLeaseOptions.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Extensions.DocumentDB
+{
+    using System;
+    using Microsoft.Azure.Documents.ChangeFeedProcessor;
+
+    /// <summary>
+    /// Configuration options for the Leases used by the trigger mechanism
+    /// </summary>
+    public class DocumentDBTriggerLeaseOptions
```

Comment:
I tried to expose it directly. The effect was that it needed to download the ChangeFeedProcessor Nuget package also on the consumer project. If a user is creating a Webjob, they would need to also download that other Nuget to customize. I felt like it was another dependency. But I can get rid of it and let the user take the dependency hit.